core-plugins/src/main/java/co/cask/hydrator/plugin/common/Properties.java

```diff
@@ -156,6 +156,18 @@
   }
 
   /**
+   * Configuration for KinesisSink
+   */
+  public static class KinesisRealtimeSink {
+    public static final String NAME = "name";
+    public static final String BODY_FIELD = "body.field";
```

Comment:
this is something I observed in the properties of other plugins. We tend to have dot.separated for cdap specific properties like "dataset.cube.properties" in cube and camelCase for external config properties eg basePath in TPFS